installation/resources/installer-config-cluster.yaml.tpl

```diff
@@ -68,54 +64,15 @@ data:
 ---
 apiVersion: v1
 kind: ConfigMap
-metadata:
-  name: core-overrides
-  namespace: kyma-installer
-  labels:
-    installer: overrides
-    component: core
-data:
-  console.cluster.headerLogoUrl: "assets/logo.svg"
-  console.cluster.headerTitle: ""
-  console.cluster.faviconUrl: "favicon.ico"
-  minio.accessKey: "admin"
-  minio.secretKey: "topSecretKey"
-  minio.resources.limits.memory: 128Mi
-  minio.resources.limits.cpu: 250m
----
-apiVersion: v1
-kind: ConfigMap
 metadata:
   name: istio-overrides
   namespace: kyma-installer
   labels:
     installer: overrides
     component: istio
 data:
-  global.proxy.includeIPRanges: "10.0.0.1/8"
-  global.proxy.excludeIPRanges: "__PROXY_EXCLUDE_IP_RANGES__"
-
-  security.enabled: "true"
```

Comment:
**security.enabled: "true"** if it's removed from both overrides then it should be moved to values.yaml in istio